vnext/Microsoft.ReactNative.Cxx/ReactCommon/TurboModuleUtils.h

```diff
@@ -0,0 +1,81 @@
+// This file is a copy from react-native NPM package to avoid dependency on folly.
+
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#pragma once
+
+#include <cassert>
+#include <string>
+
+// RNW changes: commended out the unused include that creates unnecessary dependency on folly
```

Comment:
```suggestion
// RNW changes: commented out the unused include that creates an unnecessary dependency on folly
```

Reply:
For changes like this, we should ideally be making PRs to fix in the react-native repo at the same time.

We do have a way to register files to be synced with upstream changes, but this one seems simple enough to just fix upstream.

Reply:
@asklar, good catch: I fixed it. @NickGerleman, I have submitted the PR https://github.com/facebook/react-native/pull/30672 to fix it upstream.